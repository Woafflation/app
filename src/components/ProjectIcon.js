import React from 'react'
import PropTypes from 'prop-types'
import { setDisplayName } from 'recompose'
import './ProjectIcon.css'

export const DefaultIcon = () => ''

export const ProjectIcon = ({ name, size, ticker }) => {
  if (!name) {
    return <DefaultIcon size={size} />
  }
  let imgSource = ''
  try {
    imgSource = require(`../assets/project-icons/${name
      .toString()
      .toLowerCase()
      .split(/[ /.]+/)
      .join('-')}.png`)
  } catch (e) {
    try {
      imgSource = require(`../assets/32x32/${ticker}-32.png`)
    } catch (e) {
      return <DefaultIcon size={size} />
    }
  }
  return <img width={size} alt={name} height={size} src={imgSource} />
}

ProjectIcon.propTypes = {
  size: PropTypes.number,
  name: PropTypes.string.isRequired,
  ticker: PropTypes.string
}

ProjectIcon.defaultProps = {
  size: 16,
  ticker: ''
}

export default setDisplayName('ProjectIcon')(ProjectIcon)
