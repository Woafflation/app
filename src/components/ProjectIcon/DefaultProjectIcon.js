import React from 'react'

const DefaultProjectIcon = ({ size = 16, className }) => (
  <svg
    className={className}
    width={size}
    height={size}
    viewBox='0 0 22 22'
    xmlns='http://www.w3.org/2000/svg'
  >
    <circle fill='#E7EAF3' cx='11' cy='11' r='10' />
    <path
      fill='#9FAAC4'
      d='M6.224,10.406C6.079,10.595,6,10.794,6,11.002c0,1.104,2.239,2.001,5,2.001c2.764,0,5-0.897,5-2.001 c0-0.208-0.078-0.405-0.225-0.596c-0.27,0.172-0.564,0.316-0.869,0.44c-1.051,0.419-2.43,0.659-3.906,0.659 c-1.476,0-2.856-0.239-3.907-0.659C6.787,10.723,6.493,10.578,6.224,10.406z'
    />
    <path
      fill='#9FAAC4'
      d='M6.224,12.905C6.079,13.095,6,13.291,6,13.5c0,1.104,2.239,2.002,5,2.002c2.764,0,5-0.896,5-2.002 c0-0.209-0.078-0.405-0.225-0.595c-0.27,0.168-0.564,0.313-0.869,0.436c-1.051,0.423-2.432,0.66-3.906,0.66 c-1.475,0-2.856-0.237-3.907-0.66C6.787,13.219,6.493,13.073,6.224,12.905z'
    />
    <path
      fill='#9FAAC4'
      d='M14.032,8.5c-0.021-0.006-0.035-0.014-0.054-0.021C13.291,8.204,12.238,8.001,11,8.001 c-1.238,0-2.29,0.203-2.979,0.479C8.004,8.486,7.986,8.495,7.969,8.5c0.017,0.008,0.036,0.016,0.052,0.022 C8.71,8.797,9.762,9.001,11,9.001c1.238,0,2.291-0.204,2.979-0.479C13.994,8.516,14.014,8.508,14.032,8.5z M11,10.5 c2.764,0,5-0.895,5-2c0-1.106-2.236-2-5-2c-2.761,0-5,0.895-5,2.001S8.239,10.5,11,10.5z'
    />
  </svg>
)

export default DefaultProjectIcon
