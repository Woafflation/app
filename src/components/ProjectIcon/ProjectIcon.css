.project-icon-default {
  vertical-align: middle;
  display: inline-block;
}
