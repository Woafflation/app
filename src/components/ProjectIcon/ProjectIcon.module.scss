@import '~@santiment-network/ui/mixins';

.default {
  border-radius: 50%;
  display: flex;
  align-items: center;
  justify-content: center;
  background-color: var(--background);
  fill: var(--fill);
  width: var(--size, 20px);
  height: var(--size, 20px);
  box-shadow: 0 4px 12px rgba(24, 27, 43, 0.04);
}
