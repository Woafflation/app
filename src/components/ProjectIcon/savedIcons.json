{"bitcoin": {"logoUrl": "https://production-sanbase-images.s3.amazonaws.com/uploads/logo64_bitcoin.png", "darkLogoUrl": null},"ethereum": {"logoUrl": "https://production-sanbase-images.s3.amazonaws.com/uploads/logo64_ethereum.png", "darkLogoUrl": null},"ripple": {"logoUrl": "https://production-sanbase-images.s3.amazonaws.com/uploads/logo64_ripple.png", "darkLogoUrl": null},"tether": {"logoUrl": "https://production-sanbase-images.s3.amazonaws.com/uploads/logo64_tether.png", "darkLogoUrl": null},"bitcoin-cash": {"logoUrl": "https://production-sanbase-images.s3.amazonaws.com/uploads/logo64_bitcoin-cash.png", "darkLogoUrl": null},"litecoin": {"logoUrl": "https://production-sanbase-images.s3.amazonaws.com/uploads/logo64_litecoin.png", "darkLogoUrl": null},"bitcoin-sv": {"logoUrl": "https://production-sanbase-images.s3.amazonaws.com/uploads/logo64_bitcoin-sv.png", "darkLogoUrl": null},"eos": {"logoUrl": "https://production-sanbase-images.s3.amazonaws.com/uploads/logo64_eos.png", "darkLogoUrl": null},"binance-coin": {"logoUrl": "https://production-sanbase-images.s3.amazonaws.com/uploads/logo64_binance-coin.png", "darkLogoUrl": null},"tezos": {"logoUrl": "https://production-sanbase-images.s3.amazonaws.com/uploads/logo64_tezos.png", "darkLogoUrl": null},"unus-sed-leo": {"logoUrl": "https://production-sanbase-images.s3.amazonaws.com/uploads/logo64_unus-sed-leo.png", "darkLogoUrl": null},"chainlink": {"logoUrl": "https://production-sanbase-images.s3.amazonaws.com/uploads/logo64_chainlink.png", "darkLogoUrl": null},"stellar": {"logoUrl": "https://production-sanbase-images.s3.amazonaws.com/uploads/logo64_stellar.png", "darkLogoUrl": null},"cardano": {"logoUrl": "https://production-sanbase-images.s3.amazonaws.com/uploads/logo64_cardano.png", "darkLogoUrl": null},"tron": {"logoUrl": "https://production-sanbase-images.s3.amazonaws.com/uploads/logo64_tron.png", "darkLogoUrl": null},"monero": {"logoUrl": "https://production-sanbase-images.s3.amazonaws.com/uploads/logo64_monero.png", "darkLogoUrl": null},"huobi-token": {"logoUrl": "https://production-sanbase-images.s3.amazonaws.com/uploads/logo64_huobi-token.png", "darkLogoUrl": null},"ethereum-classic": {"logoUrl": "https://production-sanbase-images.s3.amazonaws.com/uploads/logo64_ethereum-classic.png", "darkLogoUrl": null},"usd-coin": {"logoUrl": "https://production-sanbase-images.s3.amazonaws.com/uploads/logo64_usd-coin.png", "darkLogoUrl": null},"crypto-com-coin": {"logoUrl": "https://production-sanbase-images.s3.amazonaws.com/uploads/logo64_crypto-com-chain.png", "darkLogoUrl": null},"dash": {"logoUrl": "https://production-sanbase-images.s3.amazonaws.com/uploads/logo64_dash.png", "darkLogoUrl": null},"neo": {"logoUrl": "https://production-sanbase-images.s3.amazonaws.com/uploads/logo64_neo.png", "darkLogoUrl": null},"hedgetrade": {"logoUrl": "https://production-sanbase-images.s3.amazonaws.com/uploads/logo64_hedgetrade.png", "darkLogoUrl": null},"iota": {"logoUrl": "https://production-sanbase-images.s3.amazonaws.com/uploads/logo64_iota.png", "darkLogoUrl": null},"cosmos": {"logoUrl": "https://production-sanbase-images.s3.amazonaws.com/uploads/logo64_cosmos.png", "darkLogoUrl": null},"nem": {"logoUrl": "https://production-sanbase-images.s3.amazonaws.com/uploads/logo64_nem.png", "darkLogoUrl": null},"maker": {"logoUrl": "https://production-sanbase-images.s3.amazonaws.com/uploads/logo64_maker.png", "darkLogoUrl": null},"zcash": {"logoUrl": "https://production-sanbase-images.s3.amazonaws.com/uploads/logo64_zcash.png", "darkLogoUrl": null},"ontology": {"logoUrl": "https://production-sanbase-images.s3.amazonaws.com/uploads/logo64_ontology.png", "darkLogoUrl": null},"paxos-standard": {"logoUrl": "https://production-sanbase-images.s3.amazonaws.com/uploads/logo64_paxos-standard.png", "darkLogoUrl": null},"dogecoin": {"logoUrl": "https://production-sanbase-images.s3.amazonaws.com/uploads/logo64_dogecoin.png", "darkLogoUrl": null},"ftx-token": {"logoUrl": "https://production-sanbase-images.s3.amazonaws.com/uploads/logo64_ftx-token.png", "darkLogoUrl": null},"okb": {"logoUrl": "https://production-sanbase-images.s3.amazonaws.com/uploads/logo64_okb.png", "darkLogoUrl": null},"basic-attention-token": {"logoUrl": "https://production-sanbase-images.s3.amazonaws.com/uploads/logo64_basic-attention-token.png", "darkLogoUrl": null},"vechain": {"logoUrl": "https://production-sanbase-images.s3.amazonaws.com/uploads/logo64_vechain.png", "darkLogoUrl": null},"trueusd": {"logoUrl": "https://production-sanbase-images.s3.amazonaws.com/uploads/logo64_trueusd.png", "darkLogoUrl": null},"qtum": {"logoUrl": "https://production-sanbase-images.s3.amazonaws.com/uploads/logo64_qtum.png", "darkLogoUrl": null},"decred": {"logoUrl": "https://production-sanbase-images.s3.amazonaws.com/uploads/logo64_decred.png", "darkLogoUrl": null},"hedera-hashgraph": {"logoUrl": "https://production-sanbase-images.s3.amazonaws.com/uploads/logo64_hedera-hashgraph.png", "darkLogoUrl": null},"bitcoin-gold": {"logoUrl": "https://production-sanbase-images.s3.amazonaws.com/uploads/logo64_bitcoin-gold.png", "darkLogoUrl": null},"binance-usd": {"logoUrl": "https://production-sanbase-images.s3.amazonaws.com/uploads/logo64_binance-usd.png", "darkLogoUrl": null},"0x": {"logoUrl": "https://production-sanbase-images.s3.amazonaws.com/uploads/logo64_0x.png", "darkLogoUrl": null},"multi-collateral-dai": {"logoUrl": "https://production-sanbase-images.s3.amazonaws.com/uploads/logo64_multi-collateral-dai.png", "darkLogoUrl": null},"lisk": {"logoUrl": "https://production-sanbase-images.s3.amazonaws.com/uploads/logo64_lisk.png", "darkLogoUrl": null},"algorand": {"logoUrl": "https://production-sanbase-images.s3.amazonaws.com/uploads/logo64_algorand.png", "darkLogoUrl": null},"kucoin-shares": {"logoUrl": "https://production-sanbase-images.s3.amazonaws.com/uploads/logo64_kucoin-shares.png", "darkLogoUrl": null},"zb": {"logoUrl": "https://production-sanbase-images.s3.amazonaws.com/uploads/logo64_zb.png", "darkLogoUrl": null},"icon": {"logoUrl": "https://production-sanbase-images.s3.amazonaws.com/uploads/logo64_icon.png", "darkLogoUrl": null},"kyber-network": {"logoUrl": "https://production-sanbase-images.s3.amazonaws.com/uploads/logo64_kyber-network.png", "darkLogoUrl": null},"waves": {"logoUrl": "https://production-sanbase-images.s3.amazonaws.com/uploads/logo64_waves.png", "darkLogoUrl": null},"augur": {"logoUrl": "https://production-sanbase-images.s3.amazonaws.com/uploads/logo64_augur.png", "darkLogoUrl": null},"ravencoin": {"logoUrl": "https://production-sanbase-images.s3.amazonaws.com/uploads/logo64_ravencoin.png", "darkLogoUrl": null},"synthetix-network-token": {"logoUrl": "https://production-sanbase-images.s3.amazonaws.com/uploads/logo64_synthetix-network-token.png", "darkLogoUrl": null},"bitcoin-diamond": {"logoUrl": "https://production-sanbase-images.s3.amazonaws.com/uploads/logo64_bitcoin-diamond.png", "darkLogoUrl": null},"monacoin": {"logoUrl": "https://production-sanbase-images.s3.amazonaws.com/uploads/logo64_monacoin.png", "darkLogoUrl": null},"dxchain-token": {"logoUrl": "https://production-sanbase-images.s3.amazonaws.com/uploads/logo64_dxchain-token.png", "darkLogoUrl": null},"omisego": {"logoUrl": "https://production-sanbase-images.s3.amazonaws.com/uploads/logo64_omisego.png", "darkLogoUrl": null},"zencash": {"logoUrl": "https://production-sanbase-images.s3.amazonaws.com/uploads/logo64_zencash.png", "darkLogoUrl": null},"nano": {"logoUrl": "https://production-sanbase-images.s3.amazonaws.com/uploads/logo64_nano.png", "darkLogoUrl": null},"theta": {"logoUrl": "https://production-sanbase-images.s3.amazonaws.com/uploads/logo64_theta.png", "darkLogoUrl": null},"holo": {"logoUrl": "https://production-sanbase-images.s3.amazonaws.com/uploads/logo64_holo.png", "darkLogoUrl": null},"bytom": {"logoUrl": "https://production-sanbase-images.s3.amazonaws.com/uploads/logo64_bytom.png", "darkLogoUrl": null},"nexo": {"logoUrl": "https://production-sanbase-images.s3.amazonaws.com/uploads/logo64_nexo.png", "darkLogoUrl": null},"abbc-coin": {"logoUrl": "https://production-sanbase-images.s3.amazonaws.com/uploads/logo64_abbc-coin.png", "darkLogoUrl": null},"digixdao": {"logoUrl": "https://production-sanbase-images.s3.amazonaws.com/uploads/logo64_digixdao.png", "darkLogoUrl": null},"steem": {"logoUrl": "https://production-sanbase-images.s3.amazonaws.com/uploads/logo64_steem.png", "darkLogoUrl": null},"nervos-network": {"logoUrl": "https://production-sanbase-images.s3.amazonaws.com/uploads/logo64_nervos-network.png", "darkLogoUrl": null},"luna": {"logoUrl": "https://production-sanbase-images.s3.amazonaws.com/uploads/logo64_luna.png", "darkLogoUrl": null},"siacoin": {"logoUrl": "https://production-sanbase-images.s3.amazonaws.com/uploads/logo64_siacoin.png", "darkLogoUrl": null},"digibyte": {"logoUrl": "https://production-sanbase-images.s3.amazonaws.com/uploads/logo64_digibyte.png", "darkLogoUrl": null},"bittorrent": {"logoUrl": "https://production-sanbase-images.s3.amazonaws.com/uploads/logo64_bittorrent.png", "darkLogoUrl": null},"enjin-coin": {"logoUrl": "https://production-sanbase-images.s3.amazonaws.com/uploads/logo64_enjin-coin.png", "darkLogoUrl": null},"crypto-com": {"logoUrl": "https://production-sanbase-images.s3.amazonaws.com/uploads/logo64_crypto-com.png", "darkLogoUrl": null},"bytecoin-bcn": {"logoUrl": "https://production-sanbase-images.s3.amazonaws.com/uploads/logo64_bytecoin-bcn.png", "darkLogoUrl": null},"bitshares": {"logoUrl": "https://production-sanbase-images.s3.amazonaws.com/uploads/logo64_bitshares.png", "darkLogoUrl": null},"hypercash": {"logoUrl": "https://production-sanbase-images.s3.amazonaws.com/uploads/logo64_hypercash.png", "darkLogoUrl": null},"v-systems": {"logoUrl": "https://production-sanbase-images.s3.amazonaws.com/uploads/logo64_v-systems.png", "darkLogoUrl": null},"komodo": {"logoUrl": "https://production-sanbase-images.s3.amazonaws.com/uploads/logo64_komodo.png", "darkLogoUrl": null},"insolar": {"logoUrl": "https://production-sanbase-images.s3.amazonaws.com/uploads/logo64_insolar.png", "darkLogoUrl": null},"ren": {"logoUrl": "https://production-sanbase-images.s3.amazonaws.com/uploads/logo64_ren.png", "darkLogoUrl": null},"seele": {"logoUrl": "https://production-sanbase-images.s3.amazonaws.com/uploads/logo64_seele.png", "darkLogoUrl": null},"zilliqa": {"logoUrl": "https://production-sanbase-images.s3.amazonaws.com/uploads/logo64_zilliqa.png", "darkLogoUrl": null},"verge": {"logoUrl": "https://production-sanbase-images.s3.amazonaws.com/uploads/logo64_verge.png", "darkLogoUrl": null},"stasis-euro": {"logoUrl": "https://production-sanbase-images.s3.amazonaws.com/uploads/logo64_stasis-eurs.png", "darkLogoUrl": null},"status": {"logoUrl": "https://production-sanbase-images.s3.amazonaws.com/uploads/logo64_status.png", "darkLogoUrl": null},"loopring": {"logoUrl": "https://production-sanbase-images.s3.amazonaws.com/uploads/logo64_loopring.png", "darkLogoUrl": null},"ardor": {"logoUrl": "https://production-sanbase-images.s3.amazonaws.com/uploads/logo64_ardor.png", "darkLogoUrl": null},"energi": {"logoUrl": "https://production-sanbase-images.s3.amazonaws.com/uploads/logo64_energi.png", "darkLogoUrl": null},"iostoken": {"logoUrl": "https://production-sanbase-images.s3.amazonaws.com/uploads/logo64_iostoken.png", "darkLogoUrl": null},"matic-network": {"logoUrl": "https://production-sanbase-images.s3.amazonaws.com/uploads/logo64_matic-network.png", "darkLogoUrl": null},"golem-network-tokens": {"logoUrl": "https://production-sanbase-images.s3.amazonaws.com/uploads/logo64_golem-network-tokens.png", "darkLogoUrl": null},"chiliz": {"logoUrl": "https://production-sanbase-images.s3.amazonaws.com/uploads/logo64_chiliz.png", "darkLogoUrl": null},"aave": {"logoUrl": null, "darkLogoUrl": null},"aelf": {"logoUrl": "https://production-sanbase-images.s3.amazonaws.com/uploads/logo64_aelf.png", "darkLogoUrl": null},"zcoin": {"logoUrl": "https://production-sanbase-images.s3.amazonaws.com/uploads/logo64_zcoin.png", "darkLogoUrl": null},"yap-stone": {"logoUrl": "https://production-sanbase-images.s3.amazonaws.com/uploads/logo64_yap-stone.png", "darkLogoUrl": null},"aeternity": {"logoUrl": "https://production-sanbase-images.s3.amazonaws.com/uploads/logo64_aeternity.png", "darkLogoUrl": null},"blockstack": {"logoUrl": "https://production-sanbase-images.s3.amazonaws.com/uploads/logo64_blockstack.png", "darkLogoUrl": null},"waykichain": {"logoUrl": "https://production-sanbase-images.s3.amazonaws.com/uploads/logo64_waykichain.png", "darkLogoUrl": null},"quant": {"logoUrl": "https://production-sanbase-images.s3.amazonaws.com/uploads/logo64_quant.png", "darkLogoUrl": null}, "santiment": {"darkLogoUrl": "https://stage-sanbase-images.s3.amazonaws.com/uploads/9aa129b81afd692e10bb8342ef2f4b41967d04219bfed3a384d08faaabd84636_1584094057940_san.png",
      "logoUrl": "https://stage-sanbase-images.s3.amazonaws.com/uploads/logo64_santiment.png"}
}
