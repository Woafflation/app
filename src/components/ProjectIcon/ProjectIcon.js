import React from 'react'
import cx from 'classnames'
import PropTypes from 'prop-types'
import './ProjectIcon.css'

export const ProjectIcon = ({ slug, size, className }) => (
  <div
    className={cx(className, 'project-icon', `project-icon-${slug}`)}
    style={{ '--scale': size / 64, width: `${size}px`, height: `${size}px` }}
  />
)

ProjectIcon.propTypes = {
  size: PropTypes.number,
  slug: PropTypes.string.isRequired,
  className: PropTypes.string
}

ProjectIcon.defaultProps = {
  size: 20,
  className: ''
}

export default ProjectIcon
