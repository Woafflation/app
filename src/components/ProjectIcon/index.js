import React from 'react'
import PropTypes from 'prop-types'
import DefaultProjectIcon from './DefaultProjectIcon'
import './ProjectIcon.css'

export const ProjectIcon = ({ name, size, ticker, className }) => {
  if (!name) {
    return <DefaultProjectIcon size={size} className={className} />
  }
  let imgSource = ''
  try {
    imgSource = require(`../../assets/project-icons/${name
      .toString()
      .toLowerCase()
      .split(/[ /.]+/)
      .join('-')}.png`)
  } catch (e) {
    try {
      imgSource = require(`../../assets/32x32/${ticker}-32.png`)
    } catch (e) {
      return <DefaultProjectIcon size={size} className={className} />
    }
  }
  return (
    <img
      width={size}
      alt={name}
      height={size}
      src={imgSource}
      className={className}
    />
  )
}

ProjectIcon.propTypes = {
  size: PropTypes.number,
  name: PropTypes.string.isRequired,
  ticker: PropTypes.string,
  className: PropTypes.string
}

ProjectIcon.defaultProps = {
  size: 16,
  ticker: '',
  className: ''
}

export default ProjectIcon
