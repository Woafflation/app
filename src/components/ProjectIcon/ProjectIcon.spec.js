/* eslint-env jest */
import React from 'react'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import { ProjectIcon } from './ProjectIcon'

describe('ProjectIcon component', () => {
  it('(smoke) it should render correctly', () => {
    const icon = shallow(<ProjectIcon slug='bitcoin' />)
    expect(toJson(icon)).toMatchSnapshot()
  })

  it('SAN icon should render correctly', () => {
    const icon = shallow(<ProjectIcon slug='santiment' size='16' />)
    expect(toJson(icon)).toMatchSnapshot()
  })

  it('DAO.Casino icon should render correctly', () => {
    const icon = shallow(<ProjectIcon slug='monero' size='40' />)
    expect(toJson(icon)).toMatchSnapshot()
  })
})
