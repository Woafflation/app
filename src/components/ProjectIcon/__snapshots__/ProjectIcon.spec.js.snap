// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProjectIcon component (smoke) it should render correctly 1`] = `
<div
  className="project-icon project-icon-bitcoin"
  style={
    Object {
      "--scale": 0.3125,
      "height": "20px",
      "width": "20px",
    }
  }
/>
`;

exports[`ProjectIcon component DAO.Casino icon should render correctly 1`] = `
<div
  className="project-icon project-icon-monero"
  style={
    Object {
      "--scale": 0.625,
      "height": "40px",
      "width": "40px",
    }
  }
/>
`;

exports[`ProjectIcon component SAN icon should render correctly 1`] = `
<div
  className="project-icon project-icon-santiment"
  style={
    Object {
      "--scale": 0.25,
      "height": "16px",
      "width": "16px",
    }
  }
/>
`;
